recipe/meta.yaml

```
@@ -32,13 +41,26 @@ requirements:
     - numpy 1.23.*  # [py==311]
     - numpy 1.26.*  # [py>=312]
     - python-build
+    - libabseil {{ libabseil }}
+    - openssl {{ openssl }}
+    - nsync 1.24.0
+    - snappy 1.2.1
+    - double-conversion 3.1.5
+    - jsoncpp 1.9.4
+    - pybind11 2.13.6
+    - re2 2022.04.01
+    - libgrpc {{ libgrpc }}
+    - libcurl {{ libcurl }}  # [not win]
+    - zlib {{ zlib }}
+    - libprotobuf {{ libprotobuf }}
 
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    - scipy >=1.9     # [py<312]
+    - scipy >=1.10     # [py<312]
     - scipy >=1.11.1  # [py>=312]
     - ml_dtypes >=0.2.0
+    - double-conversion
```

```suggestion
    - {{ pin_compatible('double-conversion', max_pin='x.x') }}
```
Aligns with the run_exports set in cf recipe. https://github.com/conda-forge/double-conversion-feedstock/blob/main/recipe/meta.yaml#L18 
Hard to tell from double-conversion changelog which would be right: https://github.com/google/double-conversion/blob/master/Changelog